Kajabi/sage-lib
# [DSS-86]:  Storybook: remove mocks from sidebar

## Description
Removes the `mocks` directory and all related files in order to remove the mocks section from the sidebar.


## Screenshots

|  Before  |  After  |
|--------|--------|
|![Screen Shot 2022-10-10 at 2 50 27 PM](https://user-images.githubusercontent.com/1175111/194957951-e56dc7be-31c7-441b-ace2-52c20bfe0bb5.png)|![Screen Shot 2022-10-10 at 2 55 17 PM](https://user-images.githubusercontent.com/1175111/194958237-ac540cbd-8e74-4b1c-8822-c351521f25cd.png)|

## Testing in `sage-lib`
- Navigate to [Storybook](http://localhost:4100/)
- Verify Mocks section no longer displays in sidebar


## Testing in `kajabi-products`
1. (**LOW**) Removed Mocks section from Storybook sidebar. Documentation only update, no effect on KP expected.


## Related
https://kajabi.atlassian.net/browse/DSS-86
